build/includes/sdk.mk

```
@@ -108,3 +109,22 @@ ensure-build-sdk-image-base:
 # create the build image sdk if it doesn't exist
 ensure-build-sdk-image:
 	$(MAKE) build-build-sdk-image SDK_FOLDER=$(SDK_FOLDER)
+
+# Run SDK conformance Sidecar server in docker in order to run
+# SDK client test against it. Useful for test development
+SECONDS ?= 30
+run-sdk-conformance-local:
+	docker run -e "ADDRESS=" -p 59357:59357 -e "LOCAL=true" \
+	 -e "TEST=ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch" -e "TIMEOUT=$(SECONDS)" $(sidecar_tag)
+
+# Run SDK conformance test for a specific SDK_FOLDER
+SECONDS ?= 30
+run-sdk-conformance-test:
+	DOCKER_PARAMETERS="--net host" COMMAND=sdktest $(MAKE) run-sdk-command & \
+	docker run -p 59357:59357 -e "ADDRESS=" -e "LOCAL=true" \
+	 -e "TEST=ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch" -e "TIMEOUT=$(SECONDS)" --net=host  $(sidecar_tag)
+
+# Run a conformance test for all SDKs supported
+run-conformance-test:
```

```suggestion
run-sdk-conformance-tests:
```

?? WDYT? also looks similar to other ones.